Sources/TuistKit/Generator/WorkspaceGenerator.swift

```diff
@@ -59,22 +64,59 @@ final class WorkspaceGenerator: WorkspaceGenerating {
         let workspacePath = workspaceRootPath.appending(component: workspaceName)
         let workspaceData = XCWorkspaceData(children: [])
         let workspace = XCWorkspace(data: workspaceData)
+        
+        /*-----------------------------------------------------------------------------------------
```

Comment:
Do we need the `/*---` line?

Reply:
No, it's something in my Alfred snippets for my own work - not required here. Will remove.